src/main/java/com/datapipe/jenkins/vault/VaultBuildWrapper.java

```diff
@@ -148,7 +156,7 @@ protected void provideEnvironmentVariablesFromVault(Context context, Run build,
         vaultAccessor.init();
 
         for (VaultSecret vaultSecret : vaultSecrets) {
-            String path = envVars.expand(vaultSecret.getPath());
+            String path = prefixPath + envVars.expand(vaultSecret.getPath());
```

Comment:
I think we perhaps should either trim prefixPath for `/` or ensure it ends with `/`

You can use a medthod from `Hudson.Util` called `ensureEndsWith`